publish/events.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+import json
+import os
+import meetup.api
+from slackclient import SlackClient
+
+
+def getSecret(service, dirPath=os.curdir):
+    fname = '.secrets.json'
+    if not os.path.isfile(os.path.join(dirPath, fname)):
+        exit
+    with open(fname) as data:
+        data_json = json.loads(data.read())
+    if service == 'Meetup':
+        return data_json['Meetup']['api_key']
+    elif service == 'Slack':
+        return data_json['Slack']['OAuth_Access_Token']
+
+
+def getEvents(status):
+    events = meetup.api.client.GetEvents({'group_urlname': 'DevOps-Loft',
+                                          'status': status})
+    events = json.dumps(events.__dict__)
+    events = json.loads(events)
+    for event in events['results']:
+        print(event['name'])
+    meetup.api.Client(getSecret(service='Meetup'))
+    getEvents(status='past')
+
+
+def putPost(service='Slack', channel='sandbox', message='Hello World'):
```

Comment:
It's a bit misleading
You allow to pass a service name but statically post to slack?
wouldn't it be wiser to use different "client" class per service
and something of "Factory" pattern?

Reply:
Agree it is wiser, but, for MVP isn't it valuable enough? We can afford a technical debt if what right now is valuable enough.

Reply:
I agree it doesn't have to be part of MVP, but I do think that if you allow passing service, you should at least throw error is a non-supported service is passed, or, change the method name to indicate that it only handles slack and remove the service input.

Reply:
+1